test/1.4/methods/T_startup_memoized.py

```diff
@@ -21,5 +21,3 @@ def on_startup(node, meth):
 stest.expect_equal(startup_calls, expected_after_startup)
 obj = SIM_create_object('test', 'obj', [])
 stest.expect_equal(startup_calls, expected_after_startup)
-stest.expect_true(obj.runtest)
-stest.expect_equal(startup_calls, expected_after_startup)
```

Comment:
I'd complain that this changes the semantics of the test, but it doesn't really.